File: XamarinStore.iOS/Views/PrefillXamarinAccountInstructionsView.cs
using System;
using System.Drawing;
using MonoTouch.Foundation;
using MonoTouch.UIKit;
using MonoTouch.CoreGraphics;

namespace XamarinStore
{

	class PrefillXamarinAccountInstructionsView : UIView
	{
		public PrefillXamarinAccountInstructionsView ()
		{
			BackgroundColor = UIColor.White;

			var mockup = new UIImageView (UIImage.FromBundle ("fill-details-instructions-mockup")) {
				TranslatesAutoresizingMaskIntoConstraints = false
			};

			AddSubview (mockup);

			AddConstraint (NSLayoutConstraint.Create (
				mockup,
				NSLayoutAttribute.CenterY,
				NSLayoutRelation.Equal,
				this,
				NSLayoutAttribute.CenterY,
				1f, -40f
			));
		}
	}
	
}

File: F#/XamarinStore.iOS/Views/PrefillXamarinAccountInstructionsView.fs
namespace XamarinStore

open System
open System.Drawing
open MonoTouch.Foundation
open MonoTouch.UIKit
open MonoTouch.CoreGraphics

[<AllowNullLiteralAttribute>]
type PrefillXamarinAccountInstructionsView() as this =
    inherit UIView()

    do this.BackgroundColor <- UIColor.White;

       let mockup = new UIImageView (UIImage.FromBundle ("fill-details-instructions-mockup"),
                                     TranslatesAutoresizingMaskIntoConstraints = false)
       
       this.AddSubview (mockup)
       
       this.AddConstraint( NSLayoutConstraint.Create ( mockup,
                                                       NSLayoutAttribute.CenterY,
                                                       NSLayoutRelation.Equal,
                                                       this,
                                                       NSLayoutAttribute.CenterY,
                                                       1.0f, -40.0f ))